Makefile

```diff
@@ -161,6 +161,14 @@ check:
 
 .PHONY: generate
 generate:
+	@GO111MODULE=off hack/update-protobuf.sh
+	@GO111MODULE=off hack/update-codegen.sh --parallel
+	@hack/generate-parallel.sh cmd extensions pkg plugin test
+
+.PHONY: generate-sequential
```

Comment:
To prevent someone to remove the `GO111MODULE=off` part someday when we forget about this detail.

```suggestion
# We need to explicitly pass GO111MODULE=off to k8s.io/code-generator as it is significantly slower
# when it is executed without GO111MODULE=off - ref https://github.com/kubernetes/code-generator/issues/100.
.PHONY: generate-sequential
```

Reply:
As the `generate` and `generate-sequential` targets are adjacent in the Makefile, only left the comment on the `generate` target, I don't think we need 2 identical comments 5 lines across each other.